### DISCUSSION TITLE
How to add line break inside a textarea placeholder using next-ui?

### DISCUSSION THREAD

**Discussion by DaywisonSilva** [heroui-inc/heroui#2673] [Help]
![image](https://github.com/nextui-org/nextui/assets/54292958/0a777d97-2358-454f-bd3e-769f7443070a)
![image](https://github.com/nextui-org/nextui/assets/54292958/6d8e02f7-1924-4488-ad5c-6c3b564b40bc)

**Comment by alphaxek** [comment]
Hi @DaywisonSilva `&#10;` should work

![image](https://github.com/nextui-org/nextui/assets/116849110/42ef5219-1973-42fa-a2c3-195b0e45115b)

or Try hitting 'Enter' to create a new line in the text. (Not sure if its the correct way)

![image](https://github.com/nextui-org/nextui/assets/116849110/7c361512-f34c-40ec-a178-7172828bd2c6)
